CONTRIBUTING.md

```diff
@@ -2,21 +2,27 @@
 
 Contributions are welcome, and they are greatly appreciated!
 
+To contribute make sure to fork this repository, push your changes to that forked repository, then submit a pull request to this repository for your work to be merged.
+
 Every little bit helps, and credit will always be given.
-So feel free to create a [new merge/pull request][merge-link]!
 
-And make sure to follow the [guidelines](#merge-pull-request-guidelines).
+If you get stuck or you are unsure about something pop into the EA Software Engineers office hours for help.
```

Comment:
Can you capitalize Office Hours, add this link (https://docs.google.com/presentation/d/1T11dj1I3c23ZMLGWL-EYZVVJkgOv9coNR1355MWojOA/edit#slide=id.g219723c9878_0_39), and add that Office Hours are the second Monday of every month?